corehq/apps/domain/deletion.py

```diff
@@ -68,12 +69,13 @@ def execute(self, domain_name):
 
 class ModelDeletion(BaseDeletion):
 
-    def __init__(self, app_label, model_name, domain_filter_kwarg, extra_models=None):
+    def __init__(self, app_label, model_name, domain_filter_kwarg, extra_models=None, delete_kwargs=None):
```

Comment:
since `audit_action` is the only possible value for `delete_kwargs` to pass in here what about making this `audit_action=None` so that it's more explicit?